documentation/api/guides/quickstart.md

```diff
@@ -17,8 +17,12 @@ you need to run this.
 2. Clone the repository to your computer. Then switch to the cloned directory.
    If you're planning to contribute, fork the repo and clone your fork instead.
 
+   ```{note}
+   We recommend cloning with the `--filter=blob:none` flag as it dramatically reduces the filesize and download time by creating a "blobless clone". You can learn more about these [here](https://gist.github.com/leereilly/1f4ea46a01618b6e34ead76f75d0784b).
```

Comment:
Can this and the others notes in quickstart documents be wrapped?
```suggestion
   We recommend cloning with the `--filter=blob:none` flag as it dramatically
   reduces the filesize and download time by creating a "blobless clone".
   You can learn more about these [here](https://gist.github.com/leereilly/1f4ea46a01618b6e34ead76f75d0784b).
```

Reply:
@krysal sure, I'll do that. I was concerned the wrapping would be preserved on the web but apparently that isn't the case for these "admonition" blocks!